src/app/components/forms/game/description/tags/tags.ts

```diff
@@ -1,64 +1,41 @@
-import View from '!view!./tags.html?style=./tags.styl';
```

Comment:
I don't think I like this refactor...
I think we should get rid of game-description-tags component and instead put the logic in the main form and then funnel off to the app-tag-suggestions component from there.
I don't think we should have the tag-suggestion component take a TagSuggester. I think it should basically take hte props that this component takes.
If you want, inside that component, you can still make a TagSuggester model, though.

Reply:
I don't like having too much styling in the tag suggestion component. It should be scoped down to the essentials - suggesting tags. The help text block and wether it should show at all should be handled by the parent using it, but since you cant do a v-if based on a component's value you're using inside the v-if i had to refactor it so that it'd be usable before the component even gets made. That's why tag suggester is a thing, mostly.

We should talk about this, im not sure i agree hmmm